CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Add attr_accessor for `additional_services` to class `Shipcloud::Shipment` to be able to access the additional_services attribute at the shipment object.
 - Add attr_reader for `label_voucher_url` to class `Shipcloud::Shipment` to be able to read the label_voucher_url (QR Code url) attribute at the shipment object.
 - Added missing `frozen_string_literal: true` magic comments to files
+- Add attr_reader for `carrier_tracking_url` to class `Shipcloud::Shipment` to be able to read the carrier_tracking_url attribute at the shipment object.
```

Comment:
This should be moved to the `Unreleased` header, we'll move it to a new version header (probably 0.13.0) before the change is released.